api/v1alpha2/awsmachine_conversion.go

```diff
@@ -51,10 +52,13 @@ func restoreAWSMachineSpec(restored *infrav1alpha3.AWSMachineSpec, dst *infrav1a
 	}
 }
 
+func restoreAWSMachineStatus(restored *infrav1alpha3.AWSMachineStatus, dst *infrav1alpha3.AWSMachineStatus) {
+	dst.FailureReason = restored.FailureReason
```

Comment:
Shouldn't FailureMessage/FailureReason be migrated to ErrorMessage/ErrorReason in v1alpha2?

Reply:
my bad :)